python_files/generate_supplementary_plots.py

```diff
@@ -211,20 +211,53 @@
 
 
 # Image processing
+## show a run with images stitched in acquisition order pre- and post-Rosetta
+acquisition_order_viz_dir_rosetta = os.path.join(SUPPLEMENTARY_FIG_DIR, "acquisition_order_rosetta")
```

Comment:
This doesn't have anything to do with acquisition order, it's just about the rosetta pipeline. Should clean up the variable names to help disambiguate from the normalization example